exporter/prometheus/prometheus_test.go

```diff
@@ -342,3 +342,84 @@ func TestCumulativenessFromHistograms(t *testing.T) {
 		t.Fatalf("\ngot:\n%s\n\nwant:\n%s\n", got, want)
 	}
 }
+
+func TestConstLabelsIncluded(t *testing.T) {
+	key, label := "service", "spanner"
+	exporter, err := NewExporter(Options{
+		ConstLabels: prometheus.Labels{
+			key: label,
+		},
+	})
+	if err != nil {
+		t.Fatalf("failed to create prometheus exporter: %v", err)
+	}
+	view.RegisterExporter(exporter)
+
+	names := []string{"foo", "bar", "baz"}
+
+	var measures mSlice
+	for _, name := range names {
+		measures.createAndAppend("tests/"+name, name, "")
+	}
+
+	var vc vCreator
+	for _, m := range measures {
+		vc.createAndAppend(m.Name(), m.Description(), nil, m, view.Count())
+	}
+
+	if err := view.Register(vc...); err != nil {
+		t.Fatalf("failed to create views: %v", err)
+	}
+	defer view.Unregister(vc...)
+
+	view.SetReportingPeriod(time.Millisecond)
+
+	for _, m := range measures {
+		stats.Record(context.Background(), m.M(1))
```

Comment:
Since we are testing for the interaction between const labels and other labels, it'd be useful to record some tags in the context before recording these measurements e.g.
```go
ctx, _ := tag.New(context.Background(), tag.Upsert(keyExtraKey, "issue961"))
for _, m := range measures {
    stats.Record(ctx, m.M(1))
}
```

Reply:
Sure, can do!